tasks/managed/collect-index-images/collect-index-images.yaml

```diff
@@ -0,0 +1,206 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-index-images
+  labels:
```

Comment:
can you remove the labels and version? we are in the process of removing them (https://issues.redhat.com/browse/RELEASE-1676)

Reply:
Removed labels.